templates/node-cpu-hog-template.yml

```diff
@@ -0,0 +1,17 @@
+---
+variables:
+  APP_NS: default
+  APP_LABEL: "run=nginx"
+  ##only deployment is supported
+  APP_KIND: deployment 
+  TOTAL_CHAOS_DURATION: "60"
+  NODE_CPU_CORE: "2"
+  EXPERIMENT_IMAGE: litmuschaos/ansible-runner:latest
```

Comment:
wondering if we should expose only experiment image (while keeping operator and runner internal. By that I mean, keep the option for override open, don''t expose them in templates). Operator and runner are infra components of litmus - while experiment can be a dev thing. Infra can be abstracted away by default.

Reply:
Yes, @ksatchit Sounds good.